js/packages/mobile-wallet-adapter-protocol/src/startSession.ts

```diff
@@ -34,7 +34,7 @@ function getDetectionPromise() {
         const timeoutId = setTimeout(() => {
             cleanup();
             reject();
-        }, 2000);
+        }, 3000);

```

Comment:
changed the timeout for the MWA intent navigation from 2s to 3s. Helps with UX when the browser asks if the user wants to navigate away.